src/gigs-board/components/posts/PostEditor.jsx

```diff
@@ -196,8 +196,15 @@ const setLabels = (labels) => {
   });
   State.update({ labels, labelStrings });
 };
-const existingLabelStrings =
+let existingLabelStrings =
   Near.view(nearDevGovGigsContractAccountId, "get_all_labels") ?? [];
+existingLabelStrings = existingLabelStrings.filter((label) =>
+  Near.view(nearDevGovGigsContractAccountId, "is_allowed_to_use_labels", {
+    editor: context.accountId,
+    labels: [label],
+  })
+);
```

Comment:
Ouch, making 150+ view calls is extremely inefficient, especially as it does it on page load, not even at the point where users open the Post Editor. I would consider implementing a new method on the contract side similar to `is_allowed_to_use_labels`, but instead of returning a boolean, I would return a list of forbidden labels.